# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

takeOffActionGoal action_goal
takeOffActionResult action_result
takeOffActionFeedback action_feedback
